spec/ibm/cloud/sdk/vpc_spec.rb

```diff
@@ -49,32 +49,32 @@
       expect(child).to be_an_instance_of(v)
     end
   end
-end
 
-methods.each do |k, _v|
-  RSpec.describe "#{k} vpc API" do
```

Comment:
And moved this under the main Rspec describe so that it could share the vpc object